src/coreclr/jit/gentree.cpp

```diff
@@ -25989,6 +25989,18 @@ GenTree* Compiler::gtNewSimdShuffleNode(
     uint64_t value  = 0;
     simd_t   vecCns = {};
 
+    if (simdSize == 16)
```

Comment:
Presumably we can do the same for `simdSize == 8` and `ExtractVector64`?

Reply:
Probably? I've just never seen any Shuffle() on top of Vector64 at all before, so if you don't mind, I'll omit it here